src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -150,13 +162,13 @@ public async Task<IActionResult> CancelExport(string idParameter)
             return new ExportResult(response.StatusCode);
         }
 
-        private async Task<IActionResult> SendExportRequest(ExportJobType exportType, PartialDateTime since, string resourceType = null, string groupId = null)
+        private async Task<IActionResult> SendExportRequest(ExportJobType exportType, PartialDateTime since, string resourceType = null, string groupId = null, string containerName = null)
         {
-            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, exportType, resourceType, since, groupId, HttpContext.RequestAborted);
+            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, exportType, resourceType, since, groupId, containerName, HttpContext.RequestAborted);
 
             var exportResult = ExportResult.Accepted();
             exportResult.SetContentLocationHeader(_urlResolver, OperationsConstants.Export, response.JobId);
-
+            exportResult.SetDateHeader(response.JobTime.UtcDateTime.ToString("s"));
```

Comment:
I am wondering if there is a specific reason we want to include the job queuedtime information in this response. We already send that information as part of the response when export completes.

Reply:
That's fair. I was wanting a way for the user to get the folder name based only on the response from the create export request, but they really shouldn't be looking it up until they know it is done.